src/components/picture/_children/source.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import formatSrcset from "../../../utils/format-image-resizer-src";
+
+const Source = ({ height, media, resizedOptions, resizerURL, src, width, ...rest }) => {
+	if (!media || !src) return null;
+	if (resizedOptions?.auth) {
+		return (
+			<source
+				{...rest}
+				height={height}
+				media={media}
+				srcSet={formatSrcset(resizerURL.concat(src), resizedOptions, width, height)}
+				width={width}
+			/>
+		);
+	}
+	return <source {...rest} media={media} srcSet={src} height={height} width={width} />;
+};
+
+Source.defaultProps = {
+	resizedOptions: {},
+	resizerURL: "",
+	width: null,
+	height: null,
```

Comment:
Find this a little contradictory, the properties have to be `number` as noted in the propTypes but we set defaults to `null` - we can just remove the defaults for `width` and `height`

Reply:
I'm fine with this, it would just be undefined which would be falsy and work with the logic anyway.

Reply:
I'm fine as well for the same reason. Should we remove them from `Image` as well?
https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.1/src/components/image/index.jsx#L99-L100

Reply:
I've updated `source.jsx` to remove default props for `width` and `height`.